funcs_utils.php

```diff
@@ -230,6 +230,9 @@ function github_api($url, $data = [])
     $ch = curl_init($url);
     curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
     curl_setopt($ch, CURLOPT_POST, !empty($data));
+    if ($httpMethod) {
```

Comment:
This is to support PATCH + DELETE commands - code sourced from https://stackoverflow.com/a/14452435